### ISSUE TITLE
Encoding of parameters incorrect

### ISSUE THREAD

**Issue by bmc** [DanielaSfregola/twitter4s#188]
With the `TwitterStreamingClient` (v 5.3 of the library), the following code:

```scala
twitterClient.sampleStatuses(languages=Seq(Language.English))(handleTweet)
```

results in the following URL (as logged at INFO level):

```
GET https://stream.twitter.com/1.1/statuses/sample.json?filter_level=none&language=Vector%28en%29&stall_warnings=false
```

It's the `BodyEncoder.toBodyAsEncodedParams` call (as invoked by `Parameters.toString` that's the issue. It just calls `toString` on the sequence of `Language` elements.

**Comment by bmc** [comment]
(It's not clear to me that the Twitter streaming API supports multiple languages.)

**Comment by DanielaSfregola** [comment]
Thanks for reporting this bug @bmc! 

We'll try to have a look, in the meanwhile PR are always welcome. It seems like you already have identified the issue ;)

**Comment by bmc** [comment]
Was using your API for some work-related stuff. After I get past this deadline, I plan to look into submitting a PR. Right now, ENOTIME, I'm afraid...

**Comment by DanielaSfregola** [comment]
Fix released in twitter4s 5.5
